pimouse_control_node:
  ros__parameters:
    run_corridor:
      acceleration: 0.01
      deceleration: 0.02
      max_speed: 0.3
      min_speed: 0.1
      servo_target: 14.0
      servo_kp: 8.0
      servo_kd: 0.16
      servo_off_threshold: 4.0
      wall_gain: 1.0
      wall_threshold: 15.0
      near_wall_threshold: 10.0
      right_threshold: 21.0
      left_threshold: 21.0
    vision_control:
      angular_gain: 0.3
      scale_factor: 1.3
      min_neighbors: 2
